operator/src/main/resources/scripts/modelInImage.sh

```diff
@@ -260,9 +261,66 @@ function buildWDTParams_MD5() {
     OPSS_FLAGS=""
   fi
 
+  overrideWDTTimeoutValues
+
   trace "Exiting setupInventoryList"
 }
 
+function changeTimeoutProperty() {
+  trace "Entering changeTimeoutProperty"
+  sed -i "s/\($1=\).*\$/\1$2/" ${WDT_ROOT}/lib/tool.properties || exitOrLoop
+  trace "Exiting changeTimeoutProperty"
+}
+
+function overrideWDTTimeoutValues() {
+  start_trap
+  trace "Entering overrideWDTTimeoutValues"
+  # WDT defaults
+  #
+  #  connect.timeout=120000
+  #  activate.timeout=180000
+  #  deploy.timeout=180000
+  #  redeploy.timeout=180000
+  #  undeploy.timeout=180000
+  #  start.application.timeout=180000
+  #  stop.application.timeout=180000
+  #  set.server.groups.timeout=30000
+  if [ ! -z ${WDT_CONNECT_TIMEOUT} ] ; then
+    changeTimeoutProperty "connect.timeout" ${WDT_CONNECT_TIMEOUT}
+  fi
+
+  if [ ! -z ${WDT_ACTIVATE_TIMEOUT} ] ; then
+    changeTimeoutProperty "activate.timeout" ${WDT_ACTIVATE_TIMEOUT}
+  fi
+
+  if [ ! -z ${WDT_DEPLOY_TIMEOUt} ] ; then
```

Comment:
Typo: TIMEOUt   
(I thought there was a test that covers each timeout setting?  If so, why wasn't this detected?)